src/main/java/org/jenkinsci/plugins/gitclient/verifier/SshHostKeyVerificationStrategy.java

```diff
@@ -12,7 +12,7 @@ public abstract class SshHostKeyVerificationStrategy<T extends HostKeyVerifierFa
         extends AbstractDescribableImpl<SshHostKeyVerificationStrategy<T>> implements ExtensionPoint {
 
     public static final String KNOWN_HOSTS_DEFAULT =
-            Paths.get(System.getProperty("user.home"), ".ssh", "known_hosts").toString();
+            Paths.get("target", ".ssh", "known_hosts").toString();
```

Comment:
? production code not test, so the use of `target` here seems incorrect?

Reply:
because it's still marked as WIP and draft. While working on this I don't want to break my `~/.ssh content`